.travis.yml

```diff
@@ -28,7 +28,7 @@ before_install:
       fi
 
 install:
-    - sh sbin/travis.sh bootstrap
+    - sh sbin/travis.sh bootstrap --with-color-scheme-unit
```

Comment:
Maybe we just install `color-scheme-unit` anyway, it seems too much configurations to me.